baybe/recommenders/pure/nonpredictive/sampling.py

```diff
@@ -42,7 +42,6 @@ def _recommend_hybrid(
             replace=len(disc_candidates) < batch_size,
         )
 
-        cont_random.reset_index(drop=True)
```

Comment:
Why is this no longer necessary, nothing changed here?

Reply:
see the line below it was enver necessary int he first place because its simply overwritten